huntflow_api_client/models/response/dictionaries.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel, Field, PositiveInt
+
+from huntflow_api_client.models.utils import descriptions
+from huntflow_api_client.models.utils.fields import DatetimeWithTZ
+
+
+class DictionaryTaskResponsePayload(BaseModel):
+    task_id: UUID = Field(..., description="Job ID")
+
+
+class DictionaryTaskResponseMeta(BaseModel):
+    data: dict = Field(..., description="Request body")
+    account_id: int = Field(..., description=descriptions.organization_id)
+
+
+class DictionaryTaskResponse(BaseModel):
+    status: str = Field(..., description="Operation status", example="ok")
+    payload: DictionaryTaskResponsePayload
+    meta: DictionaryTaskResponseMeta
+
+
+class DictionaryUpdateResponse(DictionaryTaskResponse):
+    pass
+
+
+class DictionaryCreateResponse(DictionaryTaskResponse):
```

Comment:
Do we really need two identical classes? `DictionaryUpdateResponse` and `DictionaryCreateResponse`
Why not use one `DictionaryTaskResponse` class in both cases